src/rulesMatcher.es6

```diff
@@ -1,10 +1,15 @@
+/**
+ * Demo - https://regex101.com/r/AA4xaq/2
+ */
+const suitRegex = /^\.(?:[a-z0-9]*-)?[A-Z](?:[a-zA-Z0-9]+)\b(?!\[|:|\.is-|--)(?:-[a-zA-Z0-9]+)?/;
+
 const matchers = {
   bem({selector}) {
     return !selector.match(/(--|:)/);
   },
 
   suit({selector}) {
-    return !(selector.match(/(--|:)/) || selector.match(/\.is\-/i));
+    return suitRegex.test(selector);
```

Comment:
Is there a performance issue here though? It already checks for the `.` in the regex.